src/style.css

```
@@ -1,3 +1,86 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+:root {
+  --active-collor: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
```

Use only class selector
```suggestion
.header {
```